docs/more-pushmsg.md

```diff
@@ -33,3 +33,66 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
+    "title"=> "Hello World",
+    "message"=> "Test push From PHP cURL.",
+    "priority"=> 5,
+);
+
+$data_string = json_encode($data);
+
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array
+(
+    'Content-Type: application/json; charset=utf-8'
+);
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, $url);
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch,CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch,CURLOPT_POSTFIELDS, $data_string);
+
+$result = curl_exec($ch);
+$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+if ($code == 400)
+{
+ $message = '<strong>Bad Request</strong>';
+}
+else
+{
+ if ($code == 401)
+ {
+  $message = '<strong>Unauthorized Error - Invalid Token</strong>';
+ }
+ else
+ {
+  if ($code == 403)
+  {
+   $message = '<strong>Forbidden</strong>';
+  }
+  else
+  {
+   if ($code == 404)
+   {
+    $message = '<strong>API URL Not Found</strong>';
+   }
+   else
+   {
+    $message = '<strong>Your Message was Submitted</strong>';
```

Comment:
Only 200 status code should be accepted, in your implementation a 502 could also be considered success.

Reply:
Let me check the Official Docs clearly Give me a moments pls

Reply:
Done It Validate using HTTP status code `https://gotify.net/api-docs#/message/createMessage`

```php
$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
```